src/integration/groovy/pl/allegro/tech/build/axion/release/SimpleIntegrationTest.groovy

```diff
@@ -77,6 +77,44 @@ class SimpleIntegrationTest extends BaseIntegrationTest {
         result.output.contains('2.0.0')
         result.task(":currentVersion").outcome == TaskOutcome.SUCCESS
     }
+		
+		def "should return tag with last version if useHighestVersion is set to false"() {
```

Comment:
Shouldn't the repository just fix that automatically? I feel like most projects I work on now update these things automatically. I'll see what I can do to resolve this...